media-radio/adifmerg/adifmerg-9999.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="A small conversion and check utility for ADIF files"
+HOMEPAGE="https://github.com/oh7bf/adifmerg"
+SRC_URI="https://github.com/oh7bf/adifmerg/archive/refs/heads/master.tar.gz -> ${P}.tar.gz"
```

Comment:
This isn't right. It'll change based on whatever the latest commit is, yet you have it keyworded. But even then, it will be cached with the wrong distfile name and not re-fetched.